plantdescriptionengine/src/main/java/eu/arrowhead/core/plantdescriptionengine/pdtracker/backingstore/SqlPdStore.java

```diff
@@ -0,0 +1,161 @@
+package eu.arrowhead.core.plantdescriptionengine.pdtracker.backingstore;
+
+import eu.arrowhead.core.plantdescriptionengine.providedservices.pde_mgmt.dto.PlantDescriptionEntryDto;
+import se.arkalix.io.buf.Buffer;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+
+/**
+ * Class that reads and writes Plant Description Entries to an SQL database.
+ */
+public class SqlPdStore implements PdStore {
+
+    // Initial size of the buffer used for writing Plant Descriptions.
+    private final int INITIAL_BUFFER_SIZE = 1000;
+    private final int maxPdBytes;
+
+    // TODO: We store Plant Descriptions in their raw JSON form.
+    // In the future, we'll want to create separate tables for each subfield
+    // of a Plant Description.
+    private final String SQL_CREATE_TABLE = "CREATE TABLE IF NOT EXISTS plant_description (id INT, plant_description TEXT);";
```

Comment:
is 64kb enough to store the JSON? If not, set this to MEDIUMTEXT which can store texts up to 64MB.

Reply:
Fixed